src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+            Name:
+          <input
+           type="text"
+           name="name"
+           autocomplete="off"
+           required
+          >
+        </label>
+        <label class="form-field">
+            How old are You?
+          <input
+           type="number"
+           name="age"
+           min="1"
+           max="100"
+           value="12"
+           required
+          >
+        </label>
+        <label class="form-field">
+            Full date of birth:
+          <input
+           type="date"
+           name="dateOfbirth"
+           required
+          >
+        </label>
+        <label class="form-field">
+              I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agree"
+          required
+          >
```

Comment:
```suggestion
          I accept the term of the agreement
          <input
            type="checkbox"
            name="agree"
            required
          >
```
![image](https://user-images.githubusercontent.com/96409563/209790546-b181a978-c0e3-494a-a2de-08be2431ff4e.png)